.github/workflows/matrix.mjs

```diff
@@ -215,6 +215,14 @@ matrix.addAxis({
   ]
 });
 
+matrix.addAxis({
+  name: 'default_query_timeout',
```

Comment:
A small note: please add `title: x => x == '' ? '' : 'default_query_timeout ' + x,` so the ci job title includes default_query_timeout. Otherwise it is hard to tell what does 15 mean